### PR TITLE
Set up galaxy to work with pulp@master.

### PR DESCRIPTION
**Pull request by alikins**
Issue: #330

### CODE REVIEW

File: requirements/requirements.common.txt [hunk 1]
```diff
@@ -4,88 +4,89 @@
 #
 #    pip-compile --output-file=requirements/requirements.common.txt setup.py
 #
+git+https://github.com/pulp/pulp_ansible.git@master#egg=pulp_ansible  # via galaxy-ng (setup.py)
```
Comment (cutwater) [inline requirements/requirements.common.txt:7]: This is wrong. `requirements.txt` is used to lock versions installed in docker containers. Installing from SCM must include a fixed reference (e.g. tag or commit SHA).

Comment (alikins) [reply requirements/requirements.common.txt:7]: pip-compile really seems to want there to be a tag, but at the moment, there is no pulpcore 3.6.* tag. 
Spec'ing the latest hash from master gives warnings:

WARNING: Requested pulpcore>=3.0 from git+https://github.com/pulp/pulpcore.git@64b9fb994704691173258cd3f0fecfca70a2bff5#egg=pulpcore (from galaxy-ng (setup.py)), but installing version 3.6.0.dev0

File: setup.py [hunk 2]
```diff
@@ -60,9 +60,11 @@ def run(self):
 
 requirements = galaxy_pulp_requirements + [
     "Django~=2.2.3",
-    "pulpcore>=3.4",
-    "pulp-ansible>=0.2.0b13",
+    "pulpcore",
```
Comment (newswangerd) [inline setup.py:63]: For the RBAC work I'm doing I had to use this for my requirements

```
    "pulpcore @ git+https://github.com/pulp/pulpcore.git@master#egg=pulpcore",
    "pulp_ansible @ git+https://github.com/pulp/pulp_ansible.git@master#egg=pulp_ansible",
    "pulp_container @ git+https://github.com/pulp/pulp_container.git@master#egg=pulp_container",
```

File: requirements/requirements.common.txt (hunk 1 above)
Comment (cutwater) [reply requirements/requirements.common.txt:7]: That's likely because `pulpcore>=3.0` constraint does not include pre-release versions.

**Pull request merged**